libs/components/src/lib/menu/menu.spec.ts

```diff
@@ -719,6 +719,18 @@ describe('vwc-menu', () => {
 
 			expect(item.checked).toBe(true);
 		});
+
+		it('should handle menu items being synchronously checked after being connected', async () => {
```

Comment:
`handle` is too broad, IMO.
Is this description true?
```suggestion
		it('should gracefully fail changing menu item checked when element disconnected', async () => {
```